x/sysproxy/README.md

```diff
@@ -0,0 +1,32 @@
+# Platform support
+
+The table below summarizes system-wide support this package offers for various types of proxies on desktop platforms.
+
+| Proxy Type | Windows | MacOS | Linux |
+| --- | ----------- | ------ | ------ |
+| HTTP | Yes | Yes | Yes
+| HTTPS | Yes | Yes | Yes
+| SOCKS | No | Yes | Yes
+
+
+`SetupWebProxy` implementation in this package setups both HTTP and HTTPS proxy when they distinguished by the platform. `SetupSOCKSProxy` sets up SOCKS proxy that proxies all TCP-based connections.
+
+For example, on MacOS, if you have HTTP Proxy setup and visit a website over HTTPS, your traffic does NOT go through the proxy. 
+
+However, if only HTTPS proxy is setup, both HTTP and HTTPS requests go through the HTTPS proxy.
+
+If SOCKS is setup, both HTTP and HTTPS requests go through the proxy since it is performed at the TCP layer. SOCKS does not support UDP.
```

Comment:
SOCKS5 does support UDP. Please rephrase.

Did you mean the OS implementation doesn't support it? Did you test on all platforms?
I guess we can say that of SOCKS4 on windows, but SOCKS5 should support it.

Reply:
You are right. Sock5 does support UDP. I will update the language